tuning/data/setup_dataprocessor.py

```diff
@@ -97,7 +98,13 @@ def _process_dataconfig_file(
             )
     train_dataset = processor.process_dataset_configs(data_config.datasets)
 
-    return (train_dataset, None, data_args.dataset_text_field)
+    dataset_text_field = data_args.dataset_text_field
+    if dataset_text_field is None:
+        try:
+            dataset_text_field = get_dataset_text_field_config(data_config)
+        except KeyError:
```

Comment:
Can KeyError be raised? Or do we want something like (this would have to be changed if we decide to go with overwrite)
```suggestion
        if dataset_text_field is None:
```